dlnd/cime_config/config_component.xml

```diff
@@ -98,7 +118,7 @@
     </values>
     <group>run_component_dlnd</group>
     <file>env_run.xml</file>
-    <desc>starting year to loop data over (only used when DLND_MODE is CPLHIST or GLC_CPLHIST)</desc>
+    <desc>starting year to loop data over (only used when DLND_MODE is CPLHIST or GLC_CPLHIST or ROF_CPLHIST)</desc>
```

Comment:
Is this description correct? Elsewhere it looks like valid values of DLND_MODE are things like GCPL and RCPL, *not* CPLHIST, GLC_CPLHIST or ROF_CPLHIST.

Reply:
@billsacks - thanks for catching this. That comments is indeed incorrect. It should be just GCPL and RCPL. I definitely would recommend removing %SCPL as long as we are bringing this in.